src/foam/nanos/http/NanoRouter.java

```diff
@@ -63,7 +63,12 @@ protected void service(final HttpServletRequest req, final HttpServletResponse r
     PM       pm         = new PM(this.getClass(), serviceKey);
 
     resp.setContentType("text/html");
-
+    resp.setHeader("X-Content-Type-Options", "nosniff");
```

Comment:
Please add one line comment for each of these to explain its purpose.

Reply:
please review comments: https://github.com/foam-framework/foam2/pull/2390